src/pages/LikedVids.jsx

```diff
@@ -13,11 +13,11 @@ export function LikedVids() {
 
   return (
     <div className="flex flex-col flex-wrap px-8">
-      <h2 className="text-lg font-semibold text-gray-700  mt-4">
+      <h2 className="text-sm xs:text-lg font-semibold text-gray-700 mt-4">
         Liked Videos{" "}
       </h2>
-      <hr className="my-1 color-slate-300" />
-      {LikedVids > 0 ? (
+      <hr className="my-1 color-slate-300 mb-2" />
+      {likes.length > 0 ? (
         <>
           {likes.map(
             ({ id, title, creator, views, uploadedOn, img, avatar, about }) => (
```

Comment:
Prefer having multiple attributes on multiple lines this is just better linting and makes code more maintainable.